tests/lib/assertions/classic-model-rev1-24.json

```diff
@@ -23,7 +23,7 @@
         {
             "name": "pc-kernel",
             "type": "kernel",
-            "default-channel": "24-oem/stable"
+            "default-channel": "24-hwe/stable"
```

Comment:
I see this was updated 2 months ago:

tests: use oem instead of hwe kernel for hibryd-model test ([#14477](vscode-file://vscode-app/usr/share/code/resources/app/out/vs/code/electron-sandbox/workbench/workbench.html))

Hybrid-remodel test is failing and it is because the hwe kernel is
broken. The this change is to avoid using that kernel.

Reply:
Ah, the issue is however that it's the same revision currently in 24-hwe/stable and 24/stable - the test fails because there is no reboot happening. I think we may need to consider using different channels

Reply:
With commit 0a55c9e477496f673e60d51c4deca7ce28f0e0ca 
```
2024-11-28 09:47:56 Executing google-nested:ubuntu-24.04-64:tests/nested/manual/hybrid-remodel (nov280825-074305) (1/1)...
2024-11-28 09:50:42 Restoring google-nested:ubuntu-24.04-64:tests/nested/manual/hybrid-remodel (nov280825-074305)...
2024-11-28 09:51:16 Restoring google-nested:ubuntu-24.04-64:tests/nested/manual/ (nov280825-074305)...
2024-11-28 09:51:30 Restoring google-nested:ubuntu-24.04-64 (nov280825-074305)...
2024-11-28 09:51:31 Discarding google-nested:ubuntu-24.04-64 (nov280825-074305)...
2024-11-28 09:51:32 Successful tasks: 1
2024-11-28 09:51:32 Aborted tasks: 0
```